Resources/Prototypes/Civ14/Entities/Clothing/entities_clothing_uniform.yml

```
@@ -7527,12 +7533,17 @@
     - type: Construction
-
       graph: civ13_uniform_imperial_armor
       node: end
       cost: 5
-      material: Cloth
+      material: Leather
       time: 5
+    - type: Butcherable
+      butcheringType: Knife
+      spawned:
+      - id: MaterialLeather1
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)